src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -1182,7 +1182,7 @@ public void should_search_by_role_category_administrator_and_return_tasks_with_r
             .body("tasks.jurisdiction", everyItem(is("IA")))
             .body("tasks.case_id", hasItem(taskVariables.getCaseId()))
             .body("tasks.id", hasItem(taskId))
-            .body("tasks.role_category", everyItem(is("ADMINISTRATOR")))
+            .body("tasks.role_category", everyItem(is("ADMIN")))
             .body("total_records", equalTo(1));
 
 
```

Comment:
could we add a test that gets tasks for tasks.role_category = JUDICIAL

Reply:
we already have one test for it :
https://github.com/hmcts/wa-task-management-api/blob/5fc7db2c5a336388526a46b62ec1b4182d2d67b8/src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java#L1193